cloud/ec2_util.go

```diff
@@ -222,7 +222,17 @@ func timeTilNextEC2Payment(h *host.Host) time.Duration {
 	return time.Second
 }
 
-func usesHourlyBilling(h *host.Host) bool { return !strings.Contains(h.Distro.Arch, "linux") }
+func usesHourlyBilling(h *host.Host) bool {
+	hourly := false
```

Comment:
I think this variable is unnecessary. You can `return true` from the if statements, and then `return false` at the end.